package.json

```diff
@@ -72,5 +72,8 @@
   ],
   "types": "dist/index.d.ts",
   "exports": "./lib/index.js",
-  "type": "module"
+  "type": "module",
+  "publishConfig": {
```

Comment:
I don't think this should be propagated up to `main`. It must be removed or documented before